mindmeld/components/entity_resolver.py

```diff
@@ -567,33 +790,403 @@ def _predict_exact_match(self, entity):
                 item_value.pop("whitelist", None)
                 values.append(item_value)
 
-        return values
+        if len(values) < top_n:
```

Comment:
I am not entirely sure if this is useful? And we probably should move it to base class so we don't have duplicated code in all 3 model implementation classes?

Reply:
Sure. Incorporated this change.